assets/js/modules/idea-hub/components/dashboard/DashboardIdeasWidget/Idea.js

```diff
@@ -61,9 +68,10 @@ const Idea = ( { postEditURL, name, text, topics, buttons } ) => {
 	}, [ name ] );
 
 	const handleCreate = useCallback( () => {
-		// @TODO: Implement callback.
-		global.console.log( `Created: ${ name }` );
-	}, [ name ] );
+		setIsProcessing( true );
+		createIdeaDraftPost( { name, text, topics } );
```

Comment:
@johnPhillips we also need to wait till the new draft post is created.

Reply:
I assume you mean adding an `await` here?

Reply:
Yep.